### PR TITLE
fix validation files paths

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: reference/constraints/IsTrue.rst [hunk 1]
```diff
@@ -72,7 +72,7 @@ Then you can constrain this method with ``IsTrue``.
 
     .. code-block:: xml
 
-        <!-- src/Acme/Blogbundle/Resources/config/validation.xml -->
+        <!-- validator/validation.xml -->
```
Comment (javiereguiluz) [inline reference/constraints/IsTrue.rst]: `validator/validation.xml` -> `config/validator/validation.xml` ?

File: components/validator/resources.rst [hunk 2]
```diff
@@ -71,7 +71,7 @@ configure the locations of these files::
     use Symfony\Component\Validator\Validation;
 
     $validator = Validation::createValidatorBuilder()
-        ->addYamlMapping('config/validation.yaml')
```
Comment (wouterj) [inline components/validator/resources.rst]: I don't agree with the changes in this file: it's meant to show standalone usage of the component (a framework user will *never* write this code). So let's use `config/validation.yaml` (or even maybe `validator/validation.yaml` to be completely different)

Comment (OskarStark) [reply components/validator/resources.rst]: I updated the code parts.

**Comment by OskarStark** [comment]
Please give it another review, thanks

**Review by xabbuh — APPROVED** [review]

**Review by wouterj — APPROVED** [review]

**Pull request merged**